packages/ts/fusion-frontend/src/Authentication.ts

```diff
@@ -146,7 +166,12 @@ export async function logout(options?: LogoutOptions) {
     } catch (error) {
       // clear the token if the call fails
       clearSpringCsrfMetaTags();
-      throw error;
+
+      if (cookieExists(jwtCookieName)) {
+        deleteJWTCookie();
```

Comment:
So it only attempts to delete the cookie when some error happens, and then it discards the error if the cookie is detected.

I don’t think it’s a good idea to discard the error without even asserting its type. This would make troubleshooting harder if there is some problem (for ex., a server-side logout hander misconfiguration), but no error. Maybe it is fine to handle if we determine the error caused by a connection failure, but `doLogout` also throws if there was a non-OK response.

I’d rather suggest that we check and delete the cookie in the `finally` block, so that it is:
- removed no matter if there was an error or not
- does not discard errors

Reply:
Updated.